exercises/resistor-color/resistor_color_test.exs

```diff
@@ -0,0 +1,37 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("resistor_color.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
```

Comment:
move to `test/test_helper.exs` in the mix project

Reply:
@neenjaw moved!